src/crypto/public_key/rsa.js

```diff
@@ -223,8 +222,6 @@ export async function generate(bits, e) {
  * @async
  */
 export async function validateParams(n, e, d, p, q, u) {
-  const BigInteger = await util.getBigInteger();
```

Comment:
Same for this one and the functions below

Reply:
These functions better keep the same return signature, so that it's consistent across algorithms